designs/spot-consolidation.md

```diff
@@ -0,0 +1,105 @@
+# Spot Consolidation
```

Comment:
Should we say that this PR is going to fix #763? Since that will close the issue, we probably just want this design to relate to it, but not close the original issue when this PR gets merged in.

Reply:
I have no plans to close the issue when this design is merged.

Reply:
Right, but your description has the word "fixes" that will tell GH to close it when this is merged